l1-contracts/src.ts/deploy-process.ts

```diff
@@ -54,22 +47,33 @@ export async function initialBridgehubDeployment(
   });
   nonce++;
 
-  await deployer.deployValidatorTimelock(create2Salt, { gasPrice, nonce });
-  nonce++;
-
   await deployer.deployGovernance(create2Salt, { gasPrice, nonce });
   await deployer.deployTransparentProxyAdmin(create2Salt, { gasPrice });
   await deployer.deployBridgehubContract(create2Salt, gasPrice);
   await deployer.deployBlobVersionedHashRetriever(create2Salt, { gasPrice });
-  await deployer.deployStateTransitionManagerContract(create2Salt, extraFacets, gasPrice);
-  await deployer.setStateTransitionManagerInValidatorTimelock({ gasPrice });
 
   await deployer.deploySharedBridgeContracts(create2Salt, gasPrice);
   await deployer.deployERC20BridgeImplementation(create2Salt, { gasPrice });
   await deployer.deployERC20BridgeProxy(create2Salt, { gasPrice });
   await deployer.setParametersSharedBridge();
 }
 
+export async function deployVerifier(deployer: Deployer, gasPrice: BigNumberish, create2Salt?: string, nonce?: number) {
+  await deployer.deployVerifier(create2Salt, { gasPrice, nonce });
+}
+
+export async function deployHyperchain(
```

Comment:
Hey, I don't understand whats happening here. Maybe you want to register a hyperchain as part of a different STM in the ecosystem?

Reply:
We were thinking that chains would probably use the same STM, and use the registerHyperchain script

Reply:
Discussed offline

Regardless of if we are planning to use the same STM or not, this PR is still useful as it cleanup some logics in the script and add the ability to register a separate STM